src/UI/Seller/angular.json

```diff
@@ -37,7 +37,6 @@
               "optimization": true,
               "outputHashing": "all",
               "sourceMap": false,
-              "extractCss": true,
```

Comment:
I think this is also done on purpose so that we can load in styles separately in the release phase